discern.lic

```diff
@@ -0,0 +1,46 @@
+custom_require.call(%w[common common-travel events spellmonitor drinfomon])
+
+class AutoDiscern
+    include DRC
+    include DRCA
+
+    def initialize()
+    settings = get_settings
+    if(settings.force_discerns)
+        UserVars.discerns = {}
+    end
+    echo 'Reading spell list'
+    buffspelllist = settings.buff_spells
+    offspelllist =  settings.offensive_spells
+    ctspelllist = settings.training_spells
+     buffspelllist.each {|x| if (x[1]['use_auto_mana'])

```

Comment:
What do you think about extracting the `if` into a chained method like `buff_spell_list.select{|x| (x[1]['use_auto_mana'])}.each {|x| ...}`?

Reply:
This also applies to the other blocks